### PR TITLE
Remove BINARY_FLOAT and BINARY_DOUBLE file formats

### PR DESCRIPTION
**Pull request by pinkwah**
This feature is assumed to be unused (no uses in Azure logs) due to the specifics of the format being undocumented.

### CODE REVIEW

**Comment by pinkwah** [comment]
Jenkins test this please

File: src/clib/lib/enkf/gen_common.cpp [hunk 1]
```diff
@@ -80,43 +80,6 @@ void *gen_common_fscanf_alloc(const char *file, ecl_data_type load_data_type,
...
-            }
-        } while (!feof(stream));
-    }
-    fclose(stream);
-    *size = current_size;
-    return buffer;
-}
-
 /**
   If the load_format is binary_float or binary_double, the ASCII_type
```
Comment (ManInFez) [inline src/clib/lib/enkf/gen_common.cpp:84]: remove reference to float and double in comment

**Comment by ManInFez** [comment]
Observations also had the possibility for using binary files, and comments should be updated accordingly. see gen_obs.cpp line 138

**Review by ManInFez — APPROVED** [review]
LGTM

**Comment by pinkwah** [comment]
Jenkins test this please

**Pull request merged**